keyboards/melody96/melody96.h

```diff
@@ -25,4 +25,43 @@
 	{ K110,  K111,  K112,  K113,  K114,  K115,  K116,  K117,  K118 }  \
 }
 
+/*
+ * The layout macro for the layout of hotswap keyboards, with illustrative grid of a typical
+ * assignment.
+ * ,---------------------------------------------------------------------------
+ * |ESC|F1|F2 |F3 |F4 |F5 |F6 |F7 |F8 |F9 |F10|F11|F12|PSC|HOM|END|PGU|PGD|DEL|
+ * ,--------------------------------------------------------------------------|
+ * | ` |1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 0 | - | + | BKSPC |NLK| / | * | - |
+ * |--------------------------------------------------------------------------|
+ * | TAB | Q | W | E | R | T | Y | U | I | O | P | [ | ] |  \ | 7 | 8 | 9 |   |
+ * |----------------------------------------------------------------------| + |
+ * | CTRL  | A | S | D | F | G | H | J | K | L | ; | ' |  RET | 4 | 5 | 6 |   |
+ * |----------------------------------------------------------------------|---|
+ * | LSHIFT | Z | X | C | V | B | N | M | , | . | / |RSHFT| ↑ | 1 | 2 | 3 |   |
+ * |----------------------------------------------------------------------|ENT|
+ * |LCTL|LWIN|LALT|         SPACE         |CTL|ALT|FN | ← | ↓ | → | 0 | . |   |
+ * `---------------------------------------------------------------------------
+ */
+#define LAYOUT_HOTSWAP( \
```

Comment:
Per the naming conventions this should be `LAYOUT_hotswap`, though it might need a more specific name so as not to conflict with other layouts.

Reply:
Fixed. Leaving it at "hotswap" though, as per various other keyboards (since you can't collide between keyboards), unless you think there may be a later hotswap revision. (In which case, I'm open to suggestions about names for this!)